themes/openemr/assets/scss/_jumbotron.scss

```diff
@@ -196,6 +195,9 @@
     }
 }
 
+i.fa-heart {
```

Comment:
Instead of overriding the color here, it would probably be better to use the `text-white` utility class on the actual `<i>` where the heart is being used

Reply:
Excellent point.  I will make the change to follow the style standards for this project, and that would clean up the code.  Please let me know if you notice any other problem!  I also want to note that I do not have any official credentials for UX/UI or WCAG compliance.  I am just using the WebAIM contrast checker tool to find colors that match 4.5 contrast.  I tested screenshots of contrast effected by transparency. Feel free to double check anything.  It is possible that they somehow render with a different contrast.

Reply:
nice contribution! thank you @phillipgalbraith

Reply:
Thanks @phillipgalbraith - we appreciate the accessibility improvements.